packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/loadingaconfiguration.md

```diff
@@ -0,0 +1,60 @@
+Initialize a configuration by passing the appropriate values to `IDXClient.Configuration(issuer:clientId:clientSecret:scopes:redirectUri:)`. You can set the values directly in your code or load them from a file. This example shows loading the values from a property file in your project.
+
+First, create a property list file, such as `Okta.plist`. Next, add key-value pairs with the configuration settings for your Application Integration. For example, in the following text version of a `plist`, substitute your issuer URL for `{yourIssuerUrl}`, the client ID of your Application Integration for `{yourClientId}`, and a URI that launches your app. Set the scopes based on the access required by your app.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>issuer</key>
+    <string>{yourIssuerUrl}</string>
+    <key>clientId</key>
+    <string>{yourClientId}</string>
+    <key>redirectUri</key>
+    <string>com.sample.:/callback</string>
+    <key>scopes</key>
+    <string>openid email profile offline_access</string>
+</dict>
+</plist>
+```
+
+Next, add code to create an `IDXClient.Configuration` instance from the property list file. This example uses a decoder to read the values into an `OktaPlistContent` structure.
```

Comment:
Is adding these settings as a Plist necessary? Might it be more straightforward to just describe how to define the values in code?

Reply:
Same answer as the one I gave to Jay. From a best practices perspective these values need to be constants outside of where they're assigned. It could be done with static constants, but I picked the plist for backward compatibility with existing apps.

Is that OK?